test/react-web/client/graphql/queries.test.tsx

```diff
@@ -514,6 +514,61 @@ describe('queries', () => {
     renderer.create(<ApolloProvider client={client}><Parent /></ApolloProvider>);
   });
 
+  it('allows you to skip then unskip a query with new options (top-level syntax)', (done) => {
+    const query = gql`query people($first: Int) { allPeople(first: $first) { people { name } } }`;
+    const dataOne = { allPeople: { people: [ { name: 'Luke Skywalker' } ] } };
+    const dataTwo = { allPeople: { people: [ { name: 'Leia Skywalker' } ] } };
+    const networkInterface = mockNetworkInterface(
+      { request: { query, variables: { first: 1 } }, result: { data: dataOne } },
+      { request: { query, variables: { first: 2 } }, result: { data: dataTwo } },
+      { request: { query, variables: { first: 2 } }, result: { data: dataTwo } },
```

Comment:
I'm not sure why I need to include the same request twice here. I suspect it may be a bug in AC but I need to dig deeper.

@helfer anything immediately come to mind? The behaviour is:

1. ObservableQuery with `first: 1`, subscribed.
2. Subscription stopped
3. Variables set to `first: 2`
4. Re-subscribed.

For some reason this fires the network request twice. Any ideas?